client/components/navigation-bar.vue

```diff
@@ -15,5 +15,6 @@ export default {
   flex-wrap: wrap;
   background-color: #000;
   padding: 0 12px;
+  opacity: 0.7;
```

Comment:
this seems to result in a grayish Navigation bar
![image](https://user-images.githubusercontent.com/11838981/117197920-3a07ea00-ad9d-11eb-8e67-2f67003fc434.png)

I may push an update to address this

Reply:
Ah this was the existing behavior that I thought was intended to distinguishing what is effectively two navs, but I'm totally open to you removing!